src/vs/workbench/parts/quickopen/browser/quickopen.contribution.ts

```diff
@@ -40,6 +40,10 @@ registry.registerWorkbenchAction(new SyncActionDescriptor(QuickOpenViewPickerAct
 	primary: KeyMod.CtrlCmd | KeyCode.KEY_Q, mac: { primary: KeyMod.WinCtrl | KeyCode.KEY_Q }, linux: { primary: null }
 }), 'Quick Open View');
 
+registry.registerWorkbenchAction(new SyncActionDescriptor(ShowTasksAction, ShowTasksAction.ID, ShowTasksAction.LABEL, {
```

Comment:
I think a keyboard shortcut is a good idea though. Not sure whether we have a similar pattern to copy yet? ie another shortcut that opens the command palette to see several commands.